src/shared_modules/indexer_connector/src/monitoring.hpp

```diff
@@ -142,7 +142,7 @@ class Monitoring final
                                     }
                                     else
                                     {
-                                        value.second = false;
+                                        value.second = false; // LCOV_EXCL_LINE
```

Comment:
No, this line is executed when the `/_cat/health` API returns an incomplete response, but this is not possible because FakeOpenSearchServer always returns these values, just like the [OpenSearch API response](https://opensearch.org/docs/latest/api-reference/cat/cat-health/#response)
> I will update [this](https://github.com/wazuh/wazuh/blob/bb740e0449e914c5b71189138766c229609052bc/src/shared_modules/indexer_connector/tests/unit/fakeOpenSearchServer.hpp#L76C1-L83C1) response according to OpenSearch